Tests/TestUtils/Date+ISO8601.swift

```diff
@@ -14,6 +14,10 @@ import Foundation
 
 extension Date {
     var iso8601String: String {
```

Comment:
can we rename this one to iso8601WithMilliseconds? it confused me while reviewing the tests